team_check.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+from github3 import login
+import configparser
+import numpy as np
+
+
+### Get team members
+config = configparser.ConfigParser()
+config.read('config')
+config = config['DEFAULT']
+
+username = config['username']
+token = config['token']
+organization = config['organization']
+pairs = config['pairs']
+roster = config['roster']
+
+with open(token, 'r') as f:
+    token = f.readline().strip()
+    
+github = login(username,token=token)
+memberships = github.organization_memberships()
+course = None
+
+for membership in memberships:
+    if membership.organization.login.lower() == organization.lower():
+        course = membership.organization
+        break
+    
+teams = {team.name : team for team in list(course.teams())}
+tas = teams['TAs'].members()
+instructors = teams['Instructors'].members()
+print("Getting team members, please wait...")
+team_members = {team.name: [m.login for m in set(team.members()) - \
+    set(tas) - set(instructors)] for team in teams.values()}
```

Comment:
This line takes ~ 1 min to finish.
I guess it's a limitation of GitHub API, high concurrency is not allowed.
Other parts run less than a sec.